com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeSceneBounds.cs

```diff
@@ -102,6 +102,37 @@ public void OnBeforeSerialize()
         }
 
 #if UNITY_EDITOR
+        private int FindInflatingBrickSize(Bounds b)
+        {
+            var refVol = ProbeReferenceVolume.instance;
+            float minSizedDim = Mathf.Min(b.size.x, Mathf.Min(b.size.y, b.size.z));
+
+            float minBrickSize = refVol.MinBrickSize();
+
+            float minSideInBricks = Mathf.CeilToInt(minSizedDim / minBrickSize);
+            int subdivLevel = Mathf.CeilToInt(Mathf.Log(minSideInBricks, 3));
```

Comment:
Isn't it a FloorToInt here? because we want to take the min brick size that fits inside the min side length of the probe volume? (at least this is what I understood from your drawings yesterday)

Reply:
Yup I think you are right, in my sketches I started counting from 1 and not 0 :D Will fix up.